### ISSUE TITLE
Tarteaucitron et accessibilité

### ISSUE THREAD

**Issue by fabbea** [AmauriC/tarteaucitron.js#1163]
Bonjour, 

quand j'analyse mon site avecwww.accessibilitychecker.org pour vérifier sa compatibilité aux normes d'accessibilité, cela m'affiche cette erreur 

"Critical Issues (1) :
1. FAILING ELEMENTS
**body.home > div#tarteaucitronRoot > div#tarteaucitronIcon > button#tarteaucitronManager**
_All focusable elements must have a unique `id` to ensure that they're visible to assistive technologies."

Pouvez-vous m'en dire plus ? Bien cordialement

**Comment by nicozerr** [comment]
Bonjour,

Cela semble indiquer qu'un ID est utilisé plus d'une fois sur votre page. Auriez-vous l'URL de votre site pour y jeter un oeil ?

**Comment by AmauriC** [comment]
C'est très probablement que l'option pour afficher le petit logo tarteaucitron est activé en même temps que le "petit bandeau".

Ils partagent tous les 2 le même id.

**Comment by fabbea** [comment]
Bonjour,

tout d'abord merci pour votre réponse.
Voici l'url du site : Accueil - Valette Céline

Bien cordialement,
Béatrice et Céline



| 
| 
| 
|  |  |

 |

 |
| 
|  | 
Accueil - Valette Céline

Bienvenue sur le site internet officiel de Valette Céline, artiste auteure. Au fil des pages vous pourrez en sav...
 |

 |

 |




    Le mardi 17 octobre 2023 à 10:59:12 UTC+2, Nicolas ZERR ***@***.***> a écrit :  
 
 


Bonjour,

Cela semble indiquer qu'un ID est utilisé plus d'une fois sur votre page. Auriez-vous l'URL de votre site pour y jeter un oeil ?

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***>
  

|  | Sans virus.www.avast.com |

**Comment by AmauriC** [comment]
Merci. Il faudrait dans un premier temps mettre à jour les fichiers tarteaucitron de votre thème enfant qui commencent à dater.

Ensuite passez l'option `showAlertSmall` à `false`.

**Comment by fabbea** [comment]
Bonjour Amauri,

désolée de répondre si tard mais j'étais absente depuis quelques temps.
Je vous resitue le contexte.
Quand je teste mon site sur Accessibility Checker - ADA & WCAG Compliance (Free Scan)

| 
| 
| 
|  |  |

 |

 |
| 
|  | 
Accessibility Checker - ADA & WCAG Compliance (Free Scan)

Quickly scan your website with our free accessibility checker. Get an overview of every issue and clear instruct...
 |

 |

 |


cela m'indique l'erreur suivante :
[id] attributes on active, focusable elements are not unique   
   -    
FAILING ELEMENTS   

   
<button type="button" id="tarteaucitronManager" aria-label="Cookies (fenêtre modale)">
En conséquence j'ai fait comme vous le préconisez ci-dessous, mais l'erreur s'affiche toujours...

Bien à vous,

Béatrice et Céline



    Le mardi 17 octobre 2023 à 16:27:52 UTC+2, Amauri CHAMPEAUX ***@***.***> a écrit :  
 
 


Merci. Il faudrait dans un premier temps mettre à jour les fichiers tarteaucitron de votre thème enfant qui commencent à dater.

Ensuite passez l'option showAlertSmall à false.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***>
  

|  | Sans virus.www.avast.com |
